Committing small Azure AD sign in changes from Q3 (#12457)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/applicationsigninsummary.md

---
title: "applicationSignInDetailedSummary resource type"
description: "Represents an application sign-in summary."
localization_priority: Normal
author: "besiler"
ms.prod: "identity-and-access-reports"
doc_type: "resourcePageType"
---

# applicationSignInSummary resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Represents an application sign-in summary.

## Methods

| Method       | Return Type | Description |
|:-------------|:------------|:------------|
| [Get applicationSignInSummary](../api/applicationsigninsummary-get.md) | [applicationSignInSummary](applicationsigninsummary.md) | Read the properties and relationships of an **applicationSignInSummary** object. |

## Properties
| Property     | Type        | Description |
|:-------------|:------------|:------------|

> |appDisplayName|String|Name of the application that the user signed into.|

Reviewer: ```suggestion
|appDisplayName|String|Name of the application in which the user signed in.|
```

> |appId|String|  ID of the application that the user signed into.|

Reviewer: ```suggestion
|appId|String|  ID of the application in which the user signed in.|
```
